index.html

```diff
@@ -47,12 +47,7 @@
             src="images/bikes-for-refugees_logo.jpg"
             alt="Logo for Bikes for Refugees Scotland"
           />
```

Comment:
Hi Abed, I don't think you should have removed the `<h1>` here. It was there for accesibility of visually impaired people. Although it could not be seen in the browser, it would have been read by a screen reader.